cumulusci/robotframework/tests/salesforce/pageobjects/ObjectManagerPageObject.py

```diff
@@ -0,0 +1,144 @@
+		""" Creates a formula field by providing the field_name, formula and forumla fields"""
+		formula_locator = object_manager['input'].format("dtypeZ")
+		next_button = object_manager['button'].format("Next")
+		save_button = object_manager['button'].format("Save")
+		checkbox_option = object_manager['input'].format("fdtypeB")
+		formula_txtarea = object_manager['formula_txtarea'].format("CalculatedFormula")
+		check_syntax = object_manager['button'].format("Check Syntax")
+		
+		self.selenium.wait_until_page_contains_element(formula_locator,60)
+		self.selenium.click_element(formula_locator)
+		time.sleep(1)
+		self.selenium.click_element(next_button)
+		self.salesforce.populate_field('Field Label', field_name)
+		self.selenium.wait_until_page_contains_element(checkbox_option,60)
+		self.selenium.click_element(checkbox_option)
+		self.selenium.click_element(next_button)
+		self.selenium.wait_until_page_contains_element(formula_txtarea,60)
+		self.selenium.get_webelement(formula_txtarea).send_keys(formula)
+		self.selenium.click_element(check_syntax)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(save_button)
+		self.selenium.wait_until_location_contains("FieldsAndRelationships/view", timeout=90,
+												   message="Detail page did not load in 1 min")
+	
+	def create_lookup_field(self,field_name, related):
+		"""Creates a Lookpup field by taking in the inputs field_name and related field"""
+		lookup_locator = object_manager['input'].format("dtypeY")
+		next_button = object_manager['button'].format("Next")
+		save_button = object_manager['button'].format("Save")
+		option = object_manager['select_related_option'].format(related)
+		field_label = object_manager['input'].format("MasterLabel")
+		related = object_manager['select_related'].format("DomainEnumOrId")
+		self.selenium.wait_until_page_contains_element(lookup_locator,60)
+		self.selenium.click_element(lookup_locator)
+		time.sleep(1)
+		self.selenium.click_element(next_button)
+		self.selenium.wait_until_page_contains_element(related,60)
+		self.selenium.scroll_element_into_view(related)
+		self.selenium.get_webelement(related).click()
+		self.selenium.click_element(option)
+		time.sleep(2)
+		self.selenium.click_element(next_button)
+		self.salesforce.populate_field('Field Label', field_name)
+		self.salesforce.populate_field('Description', "This is a custom field generated during automation")
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(save_button)
+		self.selenium.wait_until_location_contains("FieldsAndRelationships/view", timeout=90,
+												   message="Detail page did not load in 1 min")
+
+	@capture_screenshot_on_error
+	def create_custom_field(self, **kwargs):
+		"""Ensure that the custom field does not exist prior and Creates a custom field based on type paramenter and the field_name
```

Comment:
I recommend trying to keep documentation line length to 80 characters or less. Really wide lines are hard to read.